curriculum/challenges/english/25-front-end-development/quiz-dictionaries-and-sets/67f412ac59601c4151b763da.md

```diff
@@ -17,439 +17,498 @@ To pass the quiz, you must correctly answer at least 18 of the 20 questions belo
+`dictionary/key/`
 
 ---
 
-Placeholder distractor 3
+`dictionary{key}`
 
 #### --answer--
 
-Placeholder answer
+`dictionary[key]`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is the correct way to update a value in a dictionary?
 
 #### --distractors--
 
-Placeholder distractor 1
+`pizza['name'] >> 'Margherita'`
 
 ---
 
-Placeholder distractor 2
+`pizza['name'] << 'Margherita'`
 
 ---
 
-Placeholder distractor 3
+`pizza['name'] == 'Margherita'`
 
 #### --answer--
 
-Placeholder answer
+`pizza['name'] = 'Margherita'`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following methods can be used to retrieve a value associated with a key?
 
 #### --distractors--
 
-Placeholder distractor 1
+`access()`
 
 ---
 
-Placeholder distractor 2
+`set()`
 
 ---
 
-Placeholder distractor 3
+`retrieve()`
 
 #### --answer--
 
-Placeholder answer
+`get()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a view object?
 
 #### --distractors--
 
-Placeholder distractor 1
+A special object used to turn strings into dictionaries.
 
 ---
 
-Placeholder distractor 2
+A way to view the content of a dictionary only if the dictionary has two or more key-value pairs.
 
 ---
 
-Placeholder distractor 3
+A special object used to turn lists into dictionaries.
 
 #### --answer--
 
-Placeholder answer
+A way to see the content of a dictionary without creating a separate copy of the data.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following methods returns a view object with all the key-value pairs in the dictionary?
 
 #### --distractors--
 
-Placeholder distractor 1
+`lists()`
 
 ---
 
-Placeholder distractor 2
+`dictionaries()`
 
 ---
 
-Placeholder distractor 3
+`collections()`
 
 #### --answer--
 
-Placeholder answer
+`items()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following methods removes all of the key-value pairs from the dictionary?
 
 #### --distractors--
 
-Placeholder distractor 1
+`empty()`
 
 ---
 
-Placeholder distractor 2
+`replace()`
 
 ---
 
-Placeholder distractor 3
+`remove()`
 
 #### --answer--
 
-Placeholder answer
+`clear()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will be the output for the following code?
+
+```py
+products = {
+    'Laptop': 990,
+    'Smartphone': 600,
+    'Tablet': 250,
+    'Headphones': 70,
+}
+
+for product in products.items():
+    print(product)
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+```py
+<'Laptop', 990>
+<'Smartphone', 600>
+<'Tablet', 250>
+<'Headphones', 70>
+```
 
 ---
 
-Placeholder distractor 2
+```py
+'Laptop', 990
+'Smartphone', 600
+'Tablet', 250
+'Headphones', 70
+```
 
 ---
 
-Placeholder distractor 3
+```py
+['Laptop', 990]
+['Smartphone', 600]
+['Tablet', 250]
+['Headphones', 70]
+```
 
 #### --answer--
 
-Placeholder answer
+```py
+('Laptop', 990)
+('Smartphone', 600)
+('Tablet', 250)
+('Headphones', 70)
```

Comment:
```suggestion
990
600
250
70
```

Reply:
When I run this in a python compiler and get this result 

<img width="1023" alt="Screenshot 2025-05-28 at 8 04 10 AM" src="https://github.com/user-attachments/assets/2c9bcbcb-c1f3-4c7b-ba0f-034bc2aaeee7" />

Reply:
Yesterday when I tested it in learn, the output was like this:

```
990
600
250
70
```

But now it's the same as you have attached :smiling_face_with_tear:
You can ignore this suggestion.